src/ProgramBuilder.ts

```diff
@@ -239,16 +238,23 @@ export class ProgramBuilder {
             if (!emitFullPaths) {
                 filePath = path.relative(cwd, filePath);
             }
-
+            let linesByFile = {};
             //load the file text
-            let fileText = await util.getFileContents(pathAbsolute);
-            let lines = util.getLines(fileText);
-
-            for (let diagnostic of sortedDiagnostics) {
-                //default the severity to error if undefined
-                let severity = typeof diagnostic.severity === 'number' ? diagnostic.severity : DiagnosticSeverity.Error;
-                //format output
-                printDiagnostic(options, severity, filePath, lines, diagnostic);
+            try {
```

Comment:
We should not be wrapping and then throwing away any runtime exceptions that happen here...that would cause even more confusion for plugin authors and developers because there wouldn't even be runtime error messages or stack traces if something went wrong.